### PR TITLE
[IMP] gitlab_api: Add documentation on how to use the API

### PR DESCRIPTION
**Pull request by WR-96**

### CODE REVIEW

File: gitlab_api.py [hunk 1]
```diff
@@ -14,6 +14,13 @@
 
 
 class GitlabAPI(object):
+    """
```
Comment (luisg123v) [inline gitlab_api.py:17]: This is a lint (quotes without any text).

The dockstring format should be:

```
"""One-line summary

Long description after a blank line, without indentation.
"""
```

Comment (WR-96) [reply gitlab_api.py]: Done :ok_hand:

File: gitlab_api.py [hunk 2]
```diff
@@ -14,6 +14,14 @@
 
 
 class GitlabAPI(object):
+    """ Custom functions using python-gitlab API
+
+    In order to use the API you need to have a config file in your $HOME, you can find an example here:
+    https://python-gitlab.readthedocs.io/en/stable/cli.html#content
+    Make sure to have a [default] section as that is the one used in this script. For the 'private_toke' you need
```
Comment (luisg123v) [inline gitlab_api.py:21]:
```suggestion
    Make sure to have a [default] section as that is the one used in this script. For the 'private_token' you need
```

Comment (WR-96) [reply gitlab_api.py]: Done :ok_hand:

**Pull request merged**